pkg/trace/config/env_test.go

```diff
@@ -253,6 +253,24 @@ func TestLoadEnv(t *testing.T) {
 		assert.Contains(cfg.ReplaceTags, rule2)
 	})
 
+	env = "DD_APM_FILTER_TAGS"
```

Comment:
We did not define this env. var. we defined:
* `DD_APM_FILTER_TAGS_REQUIRE`
* `DD_APM_FILTER_TAGS_REJECT`

Although, it's funny that this works. Let's leave the test and add the other two along.